includes/module/model/StandardCheckout.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
```

Comment:
Colocar NOTICE OF LICENSE no header do arquivo (VIDE mercadopago.php)

Reply:
Toppen